### PR TITLE
Fix Python 3 syntax errors

### PR DESCRIPTION
**Pull request by sbrun**
It fixes small Python 3 syntax errors

### CODE REVIEW

File: data/exploits/cve-2015-5287/sosreport-rhel7.py [hunk 1]
```diff
@@ -48,40 +48,40 @@ def waitpath(p):
 
 dumpdir = waitpath("/var/tmp/abrt/cc*%d" % pid)[0]
 
-print "dump directory: ", dumpdir
```
Comment (acammack-r7) [inline data/exploits/cve-2015-5287/sosreport-rhel7.py:51]: This file targets RHEL 7, which ships with Python 2.

File: lib/msf/core/modules/external/python/metasploit/cli.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from __future__ import print_function
```
Comment (acammack-r7) [inline lib/msf/core/modules/external/python/metasploit/cli.py:1]: This line must stay in place to allow us to be 2/3 compatible.

File: lib/msf/core/modules/external/python/metasploit/cli.py [hunk 3]
```diff
@@ -41,7 +41,7 @@ def parse(meta):
             choices=actions)
 
     required_group = parser.add_argument_group('required arguments')
-    for opt, props in meta['options'].items():
+    for opt, props in list(meta['options'].items()):
```
Comment (acammack-r7) [inline lib/msf/core/modules/external/python/metasploit/cli.py:44]: This `list` call is not required in a `for ... in` loop.

**Review by acammack-r7 — CHANGES_REQUESTED** [review]
Nice catch with the `except` block. The rest of these changes are not needed, though.

**Comment by sbrun** [comment]
I removed the unneeded changes. Thanks!

**Review by acammack-r7 — APPROVED** [review]

**Comment by acammack-r7** [comment]
# Release Notes
The `auxiliary/scanner/smb/impacket/wmiexec` external module has had its syntax updated to be Python 3 compatible.

**Comment by acammack-r7** [comment]
Thanks for the fix @sbrun!

**Pull request merged**